.gitignore

```diff
@@ -10,3 +10,4 @@ bookclub-deepr.knit.md
 bookclub-deepr_files
 libs
 *.html
+*.png
```

Comment:
```suggestion

```

Don't ignore images. Generally those should be checked in. You might also sometimes build them on-the-fly, but it's better to build them on your machine and check them in.